src/client.js

```diff
@@ -51,35 +51,40 @@ export default class DockerExecWebsocketClient extends EventEmitter {
     //set state, state does nothing yet
     this.state = msgcode.pause;
 
-    this.stdin = through((data) => {
+    this.stdin = through2((data, enc, cb) => {
```

Comment:
I didn't read the `through2` docs... but if you don't need `enc` or `cb` just do:
```js
this.stdin = through2(data => this.sendMessage(msgcode.stdin, data));
```

But I suspect you need to do something with cb to get flow control correct... I haven't read the docs just yet... just saying...